cppcache/src/ThinClientRegion.cpp

```diff
@@ -2852,22 +2853,25 @@ void ThinClientRegion::executeFunction(
     if (reExecuteForServ) {
       msg = new TcrMessageExecuteRegionFunction(
           new DataOutput(m_cacheImpl->createDataOutput()), func, this, args,
-          routingObj, getResult, failedNodes, timeout, m_tcrdm.get(),
+          routingObj, funcAttrs, failedNodes, timeout, m_tcrdm.get(),
           static_cast<int8_t>(1));
     } else {
       msg = new TcrMessageExecuteRegionFunction(
           new DataOutput(m_cacheImpl->createDataOutput()), func, this, args,
-          routingObj, getResult, failedNodes, timeout, m_tcrdm.get(),
+          routingObj, funcAttrs, failedNodes, timeout, m_tcrdm.get(),
           static_cast<int8_t>(0));
     }
     TcrMessageReply reply(true, m_tcrdm.get());
     // need to check
     ChunkedFunctionExecutionResponse* resultCollector(
-        new ChunkedFunctionExecutionResponse(reply, (getResult & 2) == 2, rc));
+        new ChunkedFunctionExecutionResponse(reply, funcAttrs.hasResult(), rc));
     reply.setChunkedResultHandler(resultCollector);
     reply.setTimeout(timeout);
     GfErrType err = GF_NOERR;
-    err = m_tcrdm->sendSyncRequest(*msg, reply, !(getResult & 1));
+
+    // Function failover logic is not handled in the network layer. That's why
+    // attemptFailover should be always be false when calling sendSyncRequest
+    err = m_tcrdm->sendSyncRequest(*msg, reply, false);
```

Comment:
If the comment here is true, why do we still need the attemptFailover parameter?